packages/fxa-auth-server/test/local/ip_profiling.js

```diff
@@ -67,8 +71,9 @@ function runTest(route, request, assertions) {
   }).then(assertions);
 }
 
-describe('IP Profiling', () => {
+describe('IP Profiling', function () {
   let route, accountRoutes, mockDB, mockMailer, mockRequest;
+  this.timeout(30000);
```

Comment:
When I update the `proxyquire`, I got a timeout error, where ` const { accountRoutes } = proxyquire('../../lib/routes/account', requireMocks || {});` takes more than 5 seconds to execute.

Why is it so slow? I added a timeout of 30s inside the test to allow it to pass.